.github/workflows/ci_cd.yml

```diff
@@ -23,38 +23,32 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  style:
-    name: Code style
+
+  code-style:
+    name: "Code style"
     runs-on: ubuntu-latest
     steps:
-      - name: PyAnsys code style checks
-        uses: ansys/actions/code-style@v4
+      - uses: ansys/actions/code-style@v4
         with:
           python-version: ${{ env.MAIN_PYTHON_VERSION }}
 
-  docs-style:
-    name: Documentation Style Check
+  doc-style:
+    name: "Doc style"
     runs-on: ubuntu-latest
     steps:
-      - name: PyAnsys documentation style checks
-        uses: ansys/actions/doc-style@v4
+      - uses: ansys/actions/doc-style@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
   smoke-tests:
-    name: Build and Smoke tests
+    name: "Wheelhouse for ${{ matrix.os }} and ${{ matrix.python-version }}"
     runs-on: ${{ matrix.os }}
-    needs: [style]
+    needs: code-style
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest]
```

Comment:
Question @jorgepiloto ... Why have the wheelhouses for macOS and Windows been removed?